<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgResoluciones</class>
 <widget class="QWidget" name="dlgResoluciones">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resoluciones</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTableView" name="twResoluciones"/>
         </item>
         <item>
          <widget class="QTextEdit" name="txtResolucion"/>
         </item>
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab_3">
            <attribute name="title">
             <string>Resumen</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QTextEdit" name="txtResolucionResumen">
               <property name="toolTip">
                <string>Resumen</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>Detalles</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QTreeView" name="twDetalles"/>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_9">
                 <item>
                  <widget class="QPushButton" name="btAnadirDetalles">
                   <property name="text">
                    <string>Aña&amp;dir</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btBorrarDetalles">
                   <property name="text">
                    <string>Bo&amp;rrar</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="verticalSpacer">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>Temas</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QTableView" name="twTemas"/>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_10">
                 <item>
                  <widget class="QPushButton" name="btAnadirTema">
                   <property name="text">
                    <string>Aña&amp;dir tema</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btQuitarTema">
                   <property name="text">
                    <string>Quitar tema</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="verticalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QCheckBox" name="ckEntendida">
             <property name="text">
              <string>Entendida</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="ckComprobada">
             <property name="text">
              <string>Comprobada</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="ckVolverMirar">
             <property name="text">
              <string>Volver a mirar</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="ckMandatos">
             <property name="text">
              <string>Mandatos</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btQuitarFiltros">
           <property name="text">
            <string>Quitar filtros</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolBox" name="toolBox">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>464</width>
              <height>440</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Temas</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="WidgetTemas" name="wdTemas" native="true"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pgEpigrafes">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>464</width>
              <height>440</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Epígrafes</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QTableView" name="twEpigrafesTodos"/>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QPushButton" name="btSeleccionarEpigrafe">
                   <property name="text">
                    <string>&gt;</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btDeSeleccionarEpigrafe">
                   <property name="text">
                    <string>&lt;</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btDeseleccionarTodosEpigrafes">
                   <property name="text">
                    <string>&lt;&lt;</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QTableView" name="twEpigrafesSeleccionados"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblTotalResoluciones">
         <property name="text">
          <string>Resoluciones:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblFiltro">
         <property name="text">
          <string>Filtro no activo</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btActualizarID">
         <property name="text">
          <string>Actualizar &amp;ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btActualizarResolucion">
         <property name="text">
          <string>Actualizar resolución</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetTemas</class>
   <extends>QWidget</extends>
   <header>src/widgets/widgettemas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
